src/components/Web3Status/index.tsx

```diff
@@ -173,6 +177,12 @@ function Web3StatusInner() {
         </Web3StatusConnected>
       </TraceEvent>
     )
+  } else if (!connectionReady) {
+    return (
+      <Web3StatusConnected disabled={true}>
+        <LoaderV3 size="24px" />
```

Comment:
this works well for mobile but we should consider using a pill for screenWidths > 1100px to prevent the trashing of going from the small circular loader to the full wallet pill
thrash from the spinner to full account pill
![thrashCircle](https://github.com/Uniswap/interface/assets/11512321/6b3396ba-5ef2-4cbb-a647-e31cbc4cda26)


Loader pills used on token explore for reference
![Screenshot 2023-09-20 at 9 38 21 AM](https://github.com/Uniswap/interface/assets/11512321/cabc541a-53ab-47da-b58d-65295a3a55cb)

Reply:
A loader pill would not eliminate layout shift / jank / thrashing, because the address is not displayed in a monospaced font, and the width is unknown until connection occurs. we can "cheat", but we can't necessarily get it exactly right.

Reply:
added a "cheat", lmk what you think.